terraform/cyber/cloudwatch_logs.tf

```diff
@@ -1,6 +1,6 @@
 locals {
   destination_arn = "${replace(var.csls_kinesis_destination_arn, "REGION", var.region)}"
-  log_groups      = "${formatlist("%s_%s", var.log_groups_to_ship_to_csls, var.env)}"
+  log_groups      = "${formatlist("%s_%s", concat(var.log_groups_to_ship_to_csls, list("concourse_auth_logs")), var.env)}"
```

Comment:
Can we add `"concourse_auth_logs"` to `terraform/globals.tf` instead of this `concat(...)`